include/xrpl/ledger/View.h

```diff
@@ -655,7 +655,7 @@ createPseudoAccount(
     uint256 const& pseudoOwnerKey,
     SField const& ownerField);
 
-// Returns true iff sleAcct is a pseudo-account or specific
+// Returns true if sleAcct is a pseudo-account or specific
```

Comment:
This was not a typo. I used an old shortcut from my math classes. "iff" means "if and only if". I can see where that would be confusing, though.

```suggestion
// Returns true if and only if sleAcct is a pseudo-account or specific
```

Reply:
Thanks for clarifying, Ed!